app/assets/javascripts/controllers/vm_cloud/vm_cloud_resize_form_controller.js

```diff
@@ -1,36 +1,41 @@
-ManageIQ.angular.app.controller('vmCloudResizeFormController', ['$http', '$scope', 'vmCloudResizeFormId', 'miqService', function($http, $scope, vmCloudResizeFormId, miqService) {
+ManageIQ.angular.app.controller('vmCloudResizeFormController', ['$http', '$scope', 'vmCloudResizeFormId', 'objectId', 'miqService', function($http, $scope, vmCloudResizeFormId, objectId, miqService) {
   var vm = this;
 
   var init = function() {
     vm.vmCloudModel = {
       flavor_id: null,
     };
     vm.flavors = [];
-    vm.formId = vmCloudResizeFormId;
-    vm.modelCopy = angular.copy(vm.vmCloudModel);
+    vm.vmCloudResizeformId = vmCloudResizeFormId;
```

Comment:
prefer `formId`, but if you need this name, please make sure this is `vmCloudResizeFormId` and not `...formId`

Reply:
I'll fix the capitalization of the 'F' there. With so many moving parts (and a review dragging out six months), I'd rather not mess with the name beyond that and risk breaking something that I'd forgotten about. In the context of this form (just like the reconfigure javascript form controller that I'm not touching), the ID is in fact the request, since that's the way the existing UI is already designed (again, I'm just fixing a bug here, not writing a new feature). It's a "resize instance request" that's being created or edited -- the actual VM doesn't get changed until the request is approved.

Reply:
Well, the thing is - there is a possibility you're **introducing** new bugs, just by removing the `formId` as it was before.

So... can you fix the bug without chaning the meaning of `vm.formId`?